docs/Explore Algorithms/Causal Inference/Quickstart - Synthetic difference in differences.ipynb

```diff
@@ -0,0 +1,7533 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Scalable Synthetic Difference in Differences"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "This sample notebook aims to show readers how to use SynapseML's `DiffInDiffEstimator`, `SyntheticControlEstimator` and `SyntheticDiffInDiffEstimator` to estimate the causal effect of a treatment on a particular outcome."
```

Comment:
link to your nice slides or the blog you based it on for people to understand better. Might also be able to say something simple like 

SyntheticDiffinDiff allows you to estimate the causal effect of a treatment by creating a "synthetic control group" and comparing the performance of the treatment group to this synthetic control